x-pack/plugin/sql/jdbc/src/main/java/org/elasticsearch/xpack/sql/jdbc/JdbcHttpClient.java

```diff
@@ -50,7 +49,7 @@ boolean ping(long timeoutInMs) throws SQLException {
 
     Cursor query(String sql, List<SqlTypedParamValue> params, RequestMeta meta) throws SQLException {
         int fetch = meta.fetchSize() > 0 ? meta.fetchSize() : conCfg.pageSize();
-                SqlQueryRequest sqlRequest = new SqlQueryRequest(sql, params, null, Protocol.TIME_ZONE,
+                SqlQueryRequest sqlRequest = new SqlQueryRequest(sql, params, null, conCfg.timeZone().toZoneId(),
```

Comment:
I would open up zoneId through a package protected setter - no need to go through the public timeZone() which keeps making a conversion between internal classes - that's mainly for public consumption.